bouncer/util.py

```diff
@@ -70,4 +101,35 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return {
+            "annotation_id": annotation_id,
+            "document_uri": document_uri,
+            "can_reveal_metadata": can_reveal_metadata,
+            "quote": quote,
+            "text": text
+            }
+
+
+def make_pretty_url(url):
+    """
+    Return the domain name from `url` for display.
+    """
+    try:
+        parsed_url = parse.urlparse(url)
+        pretty_url = parsed_url.netloc[:NETLOC_MAX_LENGTH]
+        if len(parsed_url.netloc) > NETLOC_MAX_LENGTH:
+            pretty_url = pretty_url + jinja2.Markup("&hellip;")
+    except:
+        return "CannotParseURL"
+    return pretty_url
+
+
+def make_boilerplate_quote(document_uri):
```

Comment:
There are multiple comments about this function, I'll add them all here instead of on each line.

1. `make_` is not a very common prefix in the Python world, `get_boilerplate_quote` should be good enough
2. The passed-in `document_uri` string could be empty as well, so the `if` statement should just be `if not document_uri` (or if you don't like `not`, then just swap the if/else bodies around.
3. Both strings ("Hypothesis annotation", and "Hypothesis annotation for ...") need to be translatable so they need to be wrapped with `_()` (e.g. `_('Hypothesis annotation')`).
4. This function needs to be unit tested to ensure it behaves correctly.